openrtb_ext/request_wrapper.go

```diff
@@ -1300,13 +1303,28 @@ func (e *ImpExt) unmarshal(extJson json.RawMessage) error {
 		}
 	}
 
+	dataJson, hasData := e.ext[dataKey]
+	if hasData {
+		e.data = &ExtImpData{}
+		if err := json.Unmarshal(dataJson, e.data); err != nil {
+			return err
+		}
+	}
+
 	tidJson, hasTid := e.ext["tid"]
 	if hasTid {
 		if err := json.Unmarshal(tidJson, &e.tid); err != nil {
 			return err
 		}
 	}
 
+	gpIdJson, hasGpId := e.ext["gpid"]
+	if hasGpId {
+		if err := json.Unmarshal(gpIdJson, &e.gpId); err != nil {
+			return err
+		}
+	}
+
```

Comment:
Can you please update `TestImpWrapperGetImpExt` in `request_wrapper_test.go` adding coverage for these two new fields?

Reply:
added test cases for gpId and data